app/view/js/help.js

```diff
@@ -6,10 +6,10 @@ Page('.help-page-index', function($page) {
     updatePage($searchInput);
   }
 
-  $page.on('keyup', '.js-search-input', $.debounce(100, function() {
+  $page.on('keyup', '.js-search-input', _.debounce(function() {
     let $searchInput = $(this);
     updatePage($searchInput);
-  }));
+  }, 100));
```

Comment:
To escape HTML on the client side, we need something like the [underscore.js](https://underscorejs.org/) library. That also provides its own `debounce` method, so we can drop the jquery plugin I was using for the purpose.